mpa/det/trainer.py

```diff
@@ -69,6 +69,7 @@ def run(self, model_cfg, model_ckpt, data_cfg, **kwargs):
 
         # Data
         datasets = [build_dataset(cfg.data.train)]
+        cfg.data.val.samples_per_gpu = cfg.data.samples_per_gpu
```

Comment:
I recommend the code below, what do you think?
```
cfg.data.val.samples_per_gpu = cfg.data.get('samples_per_gpu', 1)
```
Also, does this code matter on the cpu?

Reply:
Agree with this change, I changed it. 
In my understanding, changing batch_size can't affect the CPU/GPU training.